.snippets/code/builders/ethereum/dev-env/foundry/terminal/deploy.md

```diff
@@ -1,8 +1,8 @@
 <div id="termynal" data-termynal>
-    <span data-ty="input"><span class="file-path"></span>forge create --rpc-url https://rpc.api.moonbase.moonbeam.network \
---constructor-args 100 \
---private-key INSERT_PRIVATE_KEY \
-src/MyToken.sol:MyToken</span>
+    <span data-ty="input"><span class="file-path"></span>forge create src/MyToken.sol:MyToken \
```

Comment:
This does not work as written. It returns a Warning then outputs everything to the terminal rather than actually deploying the contract: dawnkelly@Mac foundry % forge create src/MyToken.sol:MyToken \
--rpc-url https://rpc.api.moonbase.moonbeam.network \
--account deployer \
--constructor-args 100 --broadcast
[⠊] Compiling...
No files changed, compilation skipped
Enter keystore password:
Warning: Dry run enabled, not broadcasting transaction

Reply:
Hmm I think that has to do with dry run being enabled but it looks like you have a `--broadcast` flag added 

Omitting that flag the deployment works successfully for me as written in the docs:

`forge create src/MyToken.sol:MyToken \
--rpc-url https://rpc.api.moonbase.moonbeam.network \
--account deployer \
--constructor-args 100
[⠒] Compiling...
No files changed, compilation skipped
Enter keystore password:
Deployer: 0x3B939FeaD1557C741Ff06492FD0127bd287A421e
Deployed to: 0x1e24b57E027C2Ed7a4dF9aF2D7FF52057599FE66
Transaction hash: 0x31d9e665aedda0fc3851fc2b4b9a71a9e2d884bd12f5698b03f419a66e2eefdb`